photoeditor/src/test/java/ja/burhanrashid52/photoeditor/DrawingViewApiTest.kt

```diff
@@ -26,7 +26,7 @@ internal class DrawingViewApiTest : BaseDrawingViewTest() {
             assertEquals(strokeJoin, Paint.Join.ROUND)
             assertEquals(strokeCap, Paint.Cap.ROUND)
             assertEquals(strokeWidth, ShapeBuilder.DEFAULT_SHAPE_SIZE)
-            assertEquals(alpha, ShapeBuilder.DEFAULT_SHAPE_OPACITY)
+            assertEquals(alpha, ShapeBuilder.DEFAULT_SHAPE_OPACITY ?:0)
```

Comment:
Can we make it a direct null check like this? Make more readable of what we are testing.
Thoughts @wasky ?
```kotlin
assertEquals(ShapeBuilder.DEFAULT_SHAPE_OPACITY,null)
```